lib/server.js

```diff
@@ -9,21 +12,33 @@ function serve (builder, options) {
   options = options || {}
   var server = {}
 
-  console.log('Serving on http://' + options.host + ':' + options.port + '\n')
-
   server.watcher = options.watcher || new Watcher(builder)
 
   server.app = connect().use(middleware(server.watcher))
 
-  server.http = http.createServer(server.app)
+  if (options.ssl) {
+    var sslOptions
+    try {
+      sslOptions = {
+        key: fs.readFileSync(path.join(process.cwd(), 'ssl/server.key')),
```

Comment:
`path.join(process.cwd()` here is not needed, `cwd` is implied when using `fs.*` methods